admin/repos_json_properties.md

```diff
@@ -1,27 +1,27 @@
-|Property                         |Definition                                                                                                                                                                                                                                |Allowed values                                                                                                                |Applicable repo types                                                                 |Required or optional for applicable types                                                                                    |Example: OAI (oai_dc)                                    |Example: OAI (FRDR)                         |Example: CKAN                                              |Example: CKAN                                                |Example: CKAN                                            |Example: MarkLogic                                         |Example: DataCite                                               |Example: ArcGIS                                             |Example: CSW                                                               |Example: Socrata                                                |Example: Dryad                                             |Example: Dataverse                                                  |Example: Dataverse                                                                                                                                                                                                                                                                                     |Example: Datastream                                             |Example: OpenDataSoft                                          |

```

Comment:
I'm not sure why this whole file is different (formatting?); probably ok to ignore